ui/page/dcrdex/market.go

```diff
@@ -287,8 +289,16 @@ func (pg *DEXMarketPage) OnNavigatedTo() {
 	pg.ParentWindow().ShowModal(dexPasswordModal)
 }
 
+func (pg *DEXMarketPage) isDEXReset() bool {
+	if !pg.AssetsManager.DEXCInitialized() || !pg.AssetsManager.DexClient().IsDEXPasswordSet() { // dexc was reset
+		pg.ParentNavigator().ClearStackAndDisplay(NewDEXOnboarding(pg.Load, ""))
```

Comment:
ClearStack? 🤔

We should only want to close the current page and any dex-related pages before it.

I think just `CloseCurrentPage()` and let the previous page be displayed. If that is a dex-related page, it should also check if dex has been reset and close itself.